execution/executor/src/chunk_executor/mod.rs

```diff
@@ -599,6 +599,7 @@ impl<V: VMExecutor> ChunkExecutorInner<V> {
 
         // State sync executor shouldn't have block gas limit.
         let execution_output = DoGetExecutionOutput::by_transaction_execution::<V>(
+            &V::new(),
```

Comment:
basically , here , we would need to have a single instance across the runs. 

I assume you needed to make a change here to pass module cache here. and now you can put module cache inside of V, and use instance of V here.